src/tests/integration/server.itest.js

```diff
@@ -513,6 +513,27 @@ describe('/api/monsters', () => {
       expect(res.body.results[0].name).toEqual(name);
     });
   });
+  describe('with challenge rating query', () => {
+    it('returns objects with only one provided challenge rating', async () => {
+      const res = await request(app).get('/api/monsters?challenge_rating=0.25');
+      expect(res.statusCode).toEqual(200);
```

Comment:
We might want to add an expect that makes sure what we're getting back is actually the challenge rating we expect? We can probably pick one or more at random and check. If we want to be extra pedantic, we can iterate through all and check, but that might be overkill.